Jenkinsfile.release

```
@@ -0,0 +1,176 @@
+      new Expectation('clm-server-log', 'test', '-f /var/log/nexus-iq-server/clm-server-log.log | echo $?', '0'),
+      new Expectation('audit-log', 'test', '-f /var/log/nexus-iq-server/audit.log | echo $?', '0'),
+      new Expectation('request-log', 'test', '-f /var/log/nexus-iq-server/request.log | echo $?', '0'),
+      new Expectation('stderr-log', 'test', '-f /var/log/nexus-iq-server/stderr.log | echo $?', '0'),
+      new Expectation('home-directory', 'ls', '-la /opt/sonatype | grep nexus-iq-server | awk \'\$9 !~ /^\\.*$/{print \$1,\$3,\$4,\$9}\'', 'drwxr-xr-x nexus nexus nexus-iq-server'),
+      new Expectation('start-script', 'test', '-f /opt/sonatype/nexus-iq-server/start.sh | echo $?', '0'),
+      new Expectation('start-script-has-java-opts', 'grep', '\'JAVA_OPTS\' /opt/sonatype/nexus-iq-server/start.sh | echo $?', '0'), 
+      new Expectation('work-directory', 'ls', '-la / | grep sonatype-work | awk \'\$9 !~ /^\\.*$/{print \$1,\$3,\$4,\$9}\'', 'drwxr-xr-x nexus nexus sonatype-work'), 
+      new Expectation('data-directory', 'test', '-d /sonatype-work/data | echo $?', '0'), 
+      new Expectation('config-directory', 'ls', '-la /etc | grep nexus-iq-server | awk \'\$9 !~ /^\\.*$/{print \$1,\$3,\$4,\$9}\'', 'drwxr-xr-x nexus nexus nexus-iq-server'), 
+      new Expectation('config-file', 'test', '-f /etc/nexus-iq-server | echo $?', '0') 
+    ])
+  },
+  testResults: ['**/validate-expectations-results.xml'],
+  vulnerabilityScan: {
+    nexusPolicyEvaluation(
+      iqApplication: 'docker-nexus-iq-server',
+      iqScanPatterns: [[scanPattern: "container:${env.DOCKER_IMAGE_ID}"]],
+      iqStage: 'release')
+  },
+  deploy: {
+    pushImage()
+  },
+  postDeploy: {
+    sshagent(credentials: [sonatypeZionCredentialsId()]) {
+      sh '''git config user.email "sonatype-zion@sonatype.com"
+            git tag ${env.VERSION}
+            git push origin ${env.VERSION}'''
+    }
+  },
+  onSuccess: {
+    githubStatusUpdate('success')
+  },
+  onFailure: {
+    githubStatusUpdate('failure')
+    notifyChat(currentBuild: currentBuild, env: env, room: 'iq-builds')
+  }
+)
+
+void updateIQServerVersionAndChecksum(String version, String checksum) {
+  def dockerFile = readFile(file: "${pwd()}/Dockerfile")
+  def versionRegex = /(ARG IQ_SERVER_VERSION=)(\d\.\d{1,3}\.\d\-\d{2})/
+  def shaRegex = /(ARG IQ_SERVER_SHA256=)([A-Fa-f0-9]{64})/
+
+  dockerFile = dockerFile.replaceAll(versionRegex, "\$1${version}")
+  dockerFile = dockerFile.replaceAll(shaRegex, "\$1${checksum}")
+
+  writeFile(file: "${pwd()}/Dockerfile", text: dockerFile)
+}
+
+void commitAndPushChanges(String version) {
+  runSafely 'git config --global push.default simple'
+  sonatypeZionGitConfig()
+  sshagent(credentials: [sonatypeZionCredentialsId()]) {
+    runSafely 'git add .'
+    runSafely "git diff --exit-code --cached || git commit -m 'Update to version ${version}'"
```

(to match https://github.com/sonatype/docker-nexus-iq-server/blob/master/Jenkinsfile#L139-L140)
```suggestion
    runSafely "git diff --exit-code --cached || git commit -m 'Update IQ Server to ${version}.'"
```

---

Addressed: https://github.com/sonatype/docker-nexus-iq-server/blob/CLM-22746_update_builder_node/Jenkinsfile.release#L105